rest/kadai-rest-spring/src/test/java/io/kadai/workbasket/rest/assembler/WorkbasketDefinitionRepresentationModelAssemblerTest.java

```diff
@@ -35,17 +35,17 @@
 import org.mockito.Mockito;
 import org.mockito.junit.jupiter.MockitoExtension;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.mock.mockito.SpyBean;
+import org.springframework.test.context.bean.override.mockito.MockitoSpyBean;
 
 /** Test for {@link WorkbasketDefinitionRepresentationModelAssembler}. */
 @KadaiSpringBootTest
 @ExtendWith(MockitoExtension.class)
 class WorkbasketDefinitionRepresentationModelAssemblerTest {
 
   private final WorkbasketDefinitionRepresentationModelAssembler assembler;
-  @SpyBean private WorkbasketService workbasketService;
-  @SpyBean private WorkbasketAccessItemRepresentationModelAssembler accessItemAssembler;
-  @SpyBean private WorkbasketRepresentationModelAssembler workbasketAssembler;
+  @MockitoSpyBean private WorkbasketService workbasketService;
```

Comment:
What do we need this for?

Reply:
`@SpyBean` is deprecated and will be removed in 3.6.0 in favor of `@MockitoSpyBean`.